project/ExternalCommandUtil.scala

```diff
@@ -0,0 +1,62 @@
+import java.io.{BufferedReader, File, InputStreamReader}
+import java.nio.file.{Path, Paths}
+import scala.jdk.CollectionConverters.*
+import scala.language.implicitConversions
+import scala.util.{Failure, Success, Try}
+import scala.util.control.NonFatal
+
+object ExternalCommandUtil {
+  case class ExternalCommandOutput(exitCode: Int, stdOut: Seq[String], stdErr: Seq[String]) {
```

Comment:
Given that all of the usages here use the `option`/`try` variant, it may make sense to simplify the result of `run` to always return the `Option[ExternalCommandOutput]` varaint without `stdErr` (since I don't see `stdErr` used)